src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/combination.yml

```diff
@@ -39,13 +39,13 @@ admin_products_combinations_edit_combination:
     requirements:
         combinationId: \d+
 
-admin_products_combinations_remove_combination:
-    path: /combinations/{combinationId}/remove
+admin_products_combinations_delete_combination:
+    path: /combinations/{combinationId}/delete
     methods: [DELETE]
     options:
         expose: true
     defaults:
-        _controller: PrestaShopBundle:Admin/Sell/Catalog/Product/Combination:remove
+        _controller: PrestaShopBundle:Admin/Sell/Catalog/Product/Combination:delete
```

Comment:
This doesn't seem to match the controller's action which should also be `deleteAction` no?
Also can you use the full notation please `ControllerClassFQDN::deleteAction` Since in the recent changes for sf 4.4 this short notation is not accepted any more we might as well update them correctly when we need to make some changes